### PR TITLE
property editor dropdown

### PR DESCRIPTION
**Pull request by loivsen**
Property editor dropdown

### CODE REVIEW

File: src/packages/core/components/input-dropdown/input-dropdown-list.element.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	}
+
+	#onChange(e: UUISelectEvent) {
+		e.stopPropagation();
+		if (e.target.value) this.value = e.target.value;
+		this.dispatchEvent(new CustomEvent('change', { bubbles: true, composed: true }));
+	}
+
+	render() {
+		if (!this.options) return nothing;
```
Comment (iOvergaard) [inline src/packages/core/components/input-dropdown/input-dropdown-list.element.ts:31]: Would it perhaps be better to show an empty select if there are no options, so developers can see this is happening?

File: src/packages/core/property-editor/uis/dropdown/property-editor-ui-dropdown.element.ts [hunk 2]
```diff
@@ -1,22 +1,65 @@
-import { html, customElement, property } from '@umbraco-cms/backoffice/external/lit';
-import { UmbTextStyles } from "@umbraco-cms/backoffice/style";
+import { UUISelectEvent } from '@umbraco-ui/uui';
+import type { UmbPropertyEditorConfigCollection } from '@umbraco-cms/backoffice/property-editor';
+import { html, customElement, property, state } from '@umbraco-cms/backoffice/external/lit';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
 import { UmbPropertyEditorExtensionElement } from '@umbraco-cms/backoffice/extension-registry';
+import '../../../components/input-dropdown/input-dropdown-list.element.js';
```
Comment (iOvergaard) [inline src/packages/core/property-editor/uis/dropdown/property-editor-ui-dropdown.element.ts:6]: This import path looks off somehow. Do we have an alias for this perhaps?

File: src/packages/core/components/input-dropdown/input-dropdown-list.element.ts (hunk 1 above)
Comment (loivsen) [reply src/packages/core/components/input-dropdown/input-dropdown-list.element.ts]: I think thats a good point. I changed it.

File: src/packages/core/property-editor/uis/dropdown/property-editor-ui-dropdown.element.ts (hunk 2 above)
Comment (loivsen) [reply src/packages/core/property-editor/uis/dropdown/property-editor-ui-dropdown.element.ts]: Eh, it's not even needed I just forgot to export the input element.

**Review by iOvergaard — APPROVED** [review]

**Pull request merged**